src/main/java/Calculator.java

```diff
@@ -0,0 +1,10 @@
+public class Calculator {
+
+    public static double calculate(String str) {
+        String[] split = str.split(" ");
```

Comment:
```suggestion
        String[] split = str.split(" ");
```
" " 와 같이 재사용될 수 있는 값은 상수로 선언해서 사용하는 것은 어떨까요?